syntax = "proto3";

package proto_struct;

message VssData
{
  string vss_code = 1;
  float data     = 2;
}
